packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -462,6 +465,18 @@ export default class Menu extends Component {
           {map(items, (item, index) => item && <MenuLinkItem key={index} item={item} />)}
           <li>&nbsp;</li>
           <li>&nbsp;</li>
+          {xoaStatus !== undefined && includes(xoaStatus, '✖') && +process.env.XOA_PLAN !== 5 && (
```

Comment:
The test could also be a computed which returns directly a boolean :slightly_smiling_face: